science-and-technology-for-the-built-environment.csl

```diff
@@ -0,0 +1,549 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Science And Technology for the Built Environment</title>
+    <title-short>STBE</title-short>
+    <id>http://www.zotero.org/styles/science-and-technology-for-the-built-environment</id>
+    <link href="http://www.zotero.org/styles/science-and-technology-for-the-built-environment" rel="self"/>
+    <link href="http://xp20.ashrae.org/Manuscript/InstructionsforAuthors.pdf" rel="documentation"/>
+    <author>
+      <name>Mitchell T. Paulus</name>
+      <email>paulusm14@gmail.com</email>
+    </author>    
+    <category citation-format="author-date"/>
+    <category field="psychology"/>
+    <category field="generic-base"/>
+    <updated>2016-03-14T14:54:37+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+	<issn>2374-4731</issn>
```

Comment:
https://www.ashrae.org/resources--publications/science-and-technology-for-the-built-environment shows `ISSN: 2374-4731 (Print); 2374-474X (Online)`

Please also include the online ISSN with `<eissn>2374-474X</eissn>`